### ISSUE TITLE
[Question] Does Theta method support 3-1 forecasting?

### ISSUE THREAD

**Issue by gsamaras** [unit8co/darts#882]
In the [Quickstart Notebook](https://unit8co.github.io/darts/quickstart/00-quickstart.html#Searching-for-hyper-parameters-with-the-Theta-method) we fit a ThetaModel to the train set and then forecast `len(val)` steps, i.e. we forecast as many steps as the validation set size. So the intuition here is that I learn the train set and then start forecasting from the end of it and beyond.

I am wondering if it's possible to use the Theta method to do a prediction where `m` inputs arrive (e.g. m=3) and the method predicts for 1 step? So that I don't get predictions that follow the end of the train set plot, but I somehow the method identifies that based on these 3 inputs, I am somewhere in a specific neighborhood of the train set plot, and predict the next 1 step after that neighborhood.

Let me attempt to put my thoughts in a picture:
![161049962-3489f7e5-0e7d-4f08-b1ab-980c501b9c1d](https://user-images.githubusercontent.com/12050610/161266094-46ad6e4c-657c-4ad6-8b36-d9c61be6e89d.png)

---

With LSTM for example I'd do:
```python
X = np.array([1, 3.14, 2])
inp = X.reshape((X.shape[0], X.shape[1], 1))
yhat = trained_LSTM_model.predict(inp, verbose=0)
```

Can I do something similar with the Theta method?

**Comment by hrzn** [comment]
Hi, I'm not sure I understand in full detail what you're trying to do. But in short I think the answer is no - as of today it's only possible to forecast the future of the training series the model has been trained on with Theta. There are many other models that support this in Darts, though.

**Comment by gsamaras** [comment]
@hrzn I think you are right. I am trying to put variables into play like "covariates".

For example, in the LSTM I have trained, the LSTM model has learned a function that maps a sequence of past observations as input
to an output observation. Consider a given univariate sequence:
`[10, 20, 30, 40, 50, 60, 80, 80, 90]`
We can divide the sequence into multiple input/output patterns called samples, where
three time steps are used as input and one time step is used as output for the one-step prediction that is being learned.
```
X,                       y
10, 20, 30               40
20, 30, 40               50
30, 40, 50               60
```

So I will train the LSTM model like this, and then I'll predict like that:
```
x_input = array([70, 80, 90])
x_input = x_input.reshape((1, n_steps, n_features))
yhat = model.predict(x_input)
```
where I would expect the model to predict something like `[100]`. I am wondering if I can do the same with Theta method.

Can you list me the models in Darts that achieve that please, as you mentioned above, because I am very confused, please?

**Comment by gsamaras** [comment]
> Can you list me the models in Darts that achieve that please, as you mentioned above, because I am very confused, please?

@dennisbader & @hrzn any update on this please? :) I think N-Beats is one model that achieve that from Darts, right? Is there any other please?

**Comment by hrzn** [comment]
All the models with a checkmark in the "multiple series training" on the [list of models](https://github.com/unit8co/darts#forecasting-models) support this.
